### PR TITLE
feat: CMS i18n integration

### PR DESCRIPTION
**Pull request by naren-bellala**
#### 📲 What

A description of the change.

[AB#1908](https://amido-dev.visualstudio.com/Amido-Stacks/_sprints/taskboard/Cycle%203%20-%20Front%20End%20-%20Part%202/Amido-Stacks/Cycle%203%20-%20Front%20End%20-%20Part%202/Sprint%206?workitem=1908)

#### 🤔 Why
		
To switch between languages
		
#### 🛠 How
		
More in-depth discussion of the change or implementation.

#### 👀 Evidence
		
Screenshots / external resources / links / etc.
Link to documentation updated with changes impacted in the PR
		 
#### 🕵️ How to test

Notes for QA

#### ✅ Acceptance criteria Checklist

- [ ] Code peer reviewed?
- [ ] Documentation has been updated to reflect the changes?
- [x] Passing all automated tests, including a successful deployment?
- [x] Passing any exploratory testing?
- [x] Rebased/merged with latest changes from development and re-tested?
- [x] Meeting the Coding Standards?

### CODE REVIEW

File: packages/scaffolding-cli/templates/src/ssr/components/LocaleSwitcher/index.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import {MenuItem, TextField} from "@material-ui/core"
+import {getLanguages} from "lib/contentful-api"
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/ssr/components/LocaleSwitcher/index.tsx:2]: Has this been added to the dependencies?

Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/ssr/components/LocaleSwitcher/index.tsx]: Just fixed this. Good spot.

File: packages/scaffolding-cli/templates/src/ssr/components/Header/__snapshots__/index.test.tsx.snap [hunk 2]
```diff
@@ -44,6 +44,7 @@ exports[`renders Header snapshot 1`] = `
           Yumido
         </a>
       </h2>
+      0
```
Comment (estherthetesteramido) [inline packages/scaffolding-cli/templates/src/ssr/components/Header/__snapshots__/index.test.tsx.snap:47]: What is this being rendered?

Comment (naren-bellala) [reply packages/scaffolding-cli/templates/src/ssr/components/Header/__snapshots__/index.test.tsx.snap:47]: I think it is the create button missing. Will look into it

**Pull request merged**